src/main/java/net/minecraftforge/client/event/RenderLevelLastEvent.java

```diff
@@ -10,6 +10,10 @@
 import net.minecraft.client.renderer.LevelRenderer;
 import com.mojang.math.Matrix4f;
 
+/**
+ * @see net.minecraftforge.client.event.RenderLevelStageEvent
+ */
```

Comment:
Ideally, there should be a `@deprecated` tag explaining what the closest alternative is (using `{@link}`) and why. For example, `Use {@link RenderLevelStageEvent} for more flexibility for when to render additional content, allowing better compatibility with translucency in certain graphics modes. There is no explicit replacement render stage for this event; a decision must be taken on the appropriate stage for rendering.`

The corresponding method in `ForgeHooksClient` does not need a javadoc of the same; the deprecation annotation will suffice there.